sea-orm-cli/src/commands.rs

```diff
@@ -21,6 +22,17 @@ pub async fn run_generate_command(matches: &ArgMatches<'_>) -> Result<(), Box<dy
                     .with_max_level(tracing::Level::DEBUG)
                     .with_test_writer()
                     .try_init();
+            } else {
+                let filter_layer = EnvFilter::try_new("sea_orm_codegen=info").unwrap();
+                let fmt_layer = tracing_subscriber::fmt::layer()
+                    .with_target(false)
+                    .with_level(false)
+                    .without_time();
+
+                tracing_subscriber::registry()
+                    .with(filter_layer)
+                    .with(fmt_layer)
+                    .init()
```

Comment:
Can you try below? It will fix the errors thrown on tests

```suggestion
                let _ = tracing_subscriber::registry()
                    .with(filter_layer)
                    .with(fmt_layer)
                    .try_init();
```

Reply:
It seems to good.

Thank you!